Lib/ctypes/test/test_pep3118.py

```diff
@@ -67,11 +66,10 @@ def test_endian_types(self):
                 # they are always read/write
                 self.assertFalse(v.readonly)
 
-                if v.shape:
-                    n = 1
-                    for dim in v.shape:
-                        n = n * dim
-                    self.assertEqual(n, len(v))
+                n = 1
+                for dim in v.shape:
+                    n = n * dim
+                self.assertEqual(n * v.itemsize, len(v.tobytes()))
```

Comment:
This `if` always evaluated to `False`, so the garbage code within it never ran. Fixed to be identical to the above.